src/SUMMARY.md

```
@@ -0,0 +1,26 @@
+# Summary
+
+[Quick Start 🚀](README.md)
+
+- [Frequently Asked Questions](./faq.md)
+	- [What is Mostro?](./what-is-mostro.md)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 1
Unordered list indentation

(MD007, ul-indent)